application/pom.xml

```diff
@@ -337,6 +337,22 @@
             <artifactId>Java-WebSocket</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jboss.aerogear</groupId>
+            <artifactId>aerogear-otp-java</artifactId>
+        </dependency>
+        <!-- TMP -->
```

Comment:
What does TMP mean?

Reply:
These are temporary libraries for generating a QR code for testing purposes. Will be removed when UI is ready.